src/flint/fq_nmod.jl

```diff
@@ -416,6 +416,19 @@ function add!(z::fq_nmod, x::fq_nmod, y::fq_nmod)
    return z
 end
 
+###############################################################################
+#
+#   Random functions
+#
+###############################################################################
+
+#TODO!!!
+# should use fq_nmod_randtest or fq_nmod_randtest_dense on < 2.7
+# and fq_nmod_rand on >= 2.7
```

Comment:
Would there also be a way to implement `rand` on the Julia side (i.e. using a Julia RNG) ?

Reply:
You can either call the flint functions - which is what I would do - or build the fq_nmod_t yourself.
This is want needs to happen under the hood https://github.com/wbhart/flint2/blob/trunk/fq_nmod/rand.c#L15

The problem currently is that finite fields elements in < 2.7 are quite like black boxes, so you will have a clumsy time building them using the official interface. It would also be nice to have an example where a flint rand function is called.


I think it would be good if you put it in this pr since I don't think the maintainers would merge something with my embarrassing rand(x) = 1 definition.